_episodes/twir/2022-06-06-twir-444.md

```diff
@@ -0,0 +1,202 @@
+---
+title: This Week in Rust - Issue 444
+date: 2022-06-07 01:59:33+00:00
+file: https://audio.rustacean-station.org/file/rustacean-station/2022-05-25-twir-444.mp3
+duration: "00:58:22"
+length: "28021436"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Highlights from [This Week in Rust - Issue
+444](https://this-week-in-rust.org/blog/2022/05/25/this-week-in-rust-444/). This
+week features a juicy post-mortem, open source, open hardware, and lots of news
+from around the Rust ecosystem.
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+<!--
+In this section, leave timestamped notes of the form:
+
+ - [@HH:MM:SS] - Topic at first timestamp
+ - [@HH:MM:SS] - Topic at second timestamp
+     - A link to additional material discussed during the preceding topic
+
+-->
+
+#### [@00:00] Welcome
+
+- [@00:10] - Introduction
+
+- [@00:50] - Agenda
+
+- [@01:23] - [Quote of the week](https://lobste.rs/s/wiavtb/rust_critical_retrospective#c_jkfhpb)
+  > This is the difference in approaches of the two languages. In C++ if the
+  > code is vulnerable, the blame is on the programmer. In Rust if the code is
+  > vulnerable, Rust considers it a failure of the language, and takes
+  > responsibility to stop even “bad” programmers from writing vulnerable code.
+  > I can’t stress enough how awesome it is that I can be a careless fool, and
+  > still write perfectly robust highly multi-threaded code that never crashes.
+  - [@03:09] Allen: Rust is both good and bad at marketing
+
+- [@03:30] - [Crate of the week](https://docs.rs/rustdoc-types)
+  - [@04:15] - Tim and Sean discuss parsing in [episode
+    2022-05-26](https://rustacean-station.org/episode/071-twir-442/) at 47:10
+
+#### [@05:10] Official Notices
+
+- [@05:22] - [Announcing Rust 1.61.0](https://blog.rust-lang.org/2022/05/19/Rust-1.61.0.html)
```

Comment:
You can generally skip over the official releases when discussing TWiR since we'll be going through those in a lot of depth in separate episodes.

Reply:
That point's made in the audio. We only have a cursory mention.